### ISSUE TITLE
accessibility: Chart tooltips are not keyboard accessible.

### ISSUE THREAD

**Issue by nwpappas** [recharts/recharts#3608]
- [x] I have searched the [issues](https://github.com/recharts/recharts/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://recharts.org/en-US/examples/StackedAreaChart](https://recharts.org/en-US/examples/StackedAreaChart)

### Steps to reproduce
1. Using the `tab` key, attempt to navigate to and give focus to any recharts chart.
2. When the chart has focus, use keyboard keys (tab, or arrow) to move focus to data points.
3. Verify that focus can be given through the keyboard and that tooltips appear when only using keyboard navigation

### What is expected?
Charts and active areas within the chart (those that have tooltips) should be keyboard accessible.

### What is actually happening?
Charts and active areas within the chart (those that have tooltips) are not keyboard accessible.

| Environment | Info |
|---|---|
| Recharts | v2.7.0-alpha.0 |
| React | react, react-native |
| System | Mac, Windows |
| Browser | All |

---
When any UI component are not accessible by using keyboard then user who rely on keyboard are facing difficulty and not able to perform any action and are completely block to perform any functionality on that control element.

**Comment by ckifer** [comment]
Hi, please see this section of the storybook https://release--63da8268a0da9970db6992aa.chromatic.com/?path=/docs/api-accessibility--docs which is a work in progress. 

In general you should just need to add `accessibilityLayer` to your chart and then you'll be able to keyboard navigate (see example in storybook), but it seems there is a bug with stacked charts as seen in this reproduction https://release--63da8268a0da9970db6992aa.chromatic.com/?path=/docs/api-accessibility--docs

@julianna-langston for vis

**Comment by ckifer** [comment]
Seems this is working as expected when adding `accessibilityLayer` - closing this
